networkx/algorithms/approximation/clique.py

```diff
@@ -53,30 +296,71 @@ def maximum_independent_set(G):
     NetworkXNotImplemented
         If the graph is directed or is a multigraph.
 
+    ValueError
+        If the specified `method` is not supported.
+
     Notes
     -----
-    Finds the $O(|V|/(log|V|)^2)$ apx of independent set in the worst case.
```

Comment:
Why do we want to take out the complexity report for the original method?
Is it wrong? or maybe confusing in some way?

And while we are on the topic, this new method is "linear". Linear in what, exactly?